src/diffusers/loaders/single_file_utils.py

```diff
@@ -257,6 +257,39 @@ def fetch_ldm_config_and_checkpoint(
     return original_config, checkpoint
 
 
+def load_single_file_model_checkpoint(pretrained_model_link_or_path, **kwargs):
```

Comment:
I find it confusing with `fetch_ldm_config_and_checkpoint()`: 

https://github.com/huggingface/diffusers/blob/a1cb106459da4c595d22c04e026d7169d8dcfd2b/src/diffusers/loaders/single_file_utils.py#L221

Can't we repurpose `fetch_ldm_config_and_checkpoint()` here? Code looks almost the same to me.